xpano/main.cc

```diff
@@ -108,7 +108,9 @@ int main(int /*unused*/, char** argv) {
 
   // Application specific
   auto backend = xpano::gui::backends::Sdl{renderer};
-  auto license_texts = xpano::utils::LoadTexts(argv[0], xpano::kLicensePath);
+
+  std::future<std::vector<xpano::utils::Text>> license_texts = std::async(std::launch::async, xpano::utils::LoadTexts, argv[0], xpano::kLicensePath);
```

Comment:
Please format all your code with clang-format, I added instructions in #25 

If you sync with the `main` branch, a workflow will automatically check the formatting after you commit